frameworks/g-cloud-9/questions/services/securityGovernanceStandards.yml

```diff
@@ -15,8 +15,10 @@ type: checkboxes
 options:
   - label: CSA CCM v3.0
     value: csa_ccm
+    filter_label: CSA CCM version 3.0
```

Comment:
Probably better to edit the label itself, if we're saying the content deserves to be changed. Otherwise we have two different versions of the same thing.